### PR TITLE
NRPT-144: Template updates

### PR DESCRIPTION
**Pull request by dhlevi**
Updating templates to newer versions, nodejs 10. Cleanup of some old files. Templates might still need some adjustments, they were pretty out of date.

See ticket [NRPT-144](https://bcmines.atlassian.net/browse/NRPT-144)

### CODE REVIEW

File: openshift/templates/nginx-runtime/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,34 @@
...
+# ---------------------------------------------------------------------------------
+RUN chmod -R g+rwx $STI_SCRIPTS_PATH
+RUN chmod g+rw /var/cache/nginx \
+               /var/run \
+               /var/log/nginx \
+               /etc/nginx/publicServerEnvironmentSettings.js \
+               /etc/nginx/nginx.conf \
+               /tmp
+# =================================================================================
+RUN chmod 777 /run
```
Comment (MaxWardle) [inline openshift/templates/nginx-runtime/Dockerfile:32]: Have we checked that these changes aren't breaking? It looks like the right permissions from epic, but good to be sure. I remember on epic that this image doesn't trigger a rebuild unless you manually do it. When it finally rebuilt and didn't work we had forgotten about this change.

Comment (dhlevi) [reply openshift/templates/nginx-runtime/Dockerfile:32]: Nope, I haven't checked yet. I just wanted to get some eyes on the template updates before I started breaking environments completely.

Comment (MaxWardle) [reply openshift/templates/nginx-runtime/Dockerfile:32]: haha okay well as long as we remember to rebuild this image after we merge this. The rest of this pr looks good to me, just thought I would make a note of this so we don't run into the same problem again down the line.

File: openshift/templates/angular-on-nginx/angular-on-nginx-build.json [hunk 2]
```diff
@@ -91,14 +93,14 @@
...
     {
-      "name": "GIT_SOURCE_URL",
-      "displayName": "GIT Source Repo URL",
-      "description": "A GIT URL to your source code.",
+      "name": "ANGULAR_BUILDER_IMAGE",
+      "displayName": "Angular Builder Image",
+      "description": "The image that we compiled our code in.",
       "required": true,
-      "value": "https://github.com/bcgov/angular-scaffold"
+      "value": "angular-builder"
```
Comment (mtCarto) [inline openshift/templates/angular-on-nginx/angular-on-nginx-build.json:103]: Let's point this to the bcmi template as default

**Pull request merged**